xarray/util/generate_ops.py

```diff
@@ -218,6 +218,8 @@ def unops() -> list[OpsType]:
 # type-ignores end up in the wrong line :/
 
 ops_info = {}
+# TODO add inplace ops for DataTree?
+ops_info["DataTreeOpsMixin"] = binops(other_type="DtCompatible") + unops()
```

Comment:
That's nice that this just works out of the box :)

Reply:
Isn't it! I was surprised how easy this was.